### ISSUE TITLE
3.4.2 bug Cannot read property 'updateOn' of undefined

### ISSUE THREAD

**Issue by stephennancekivell** [ruhley/angular-color-picker#167]
Hi mate, somethings not right with this latest release. I get this error. The tests seem to pass though. (congrats for getting travis to work! 🤠)

I think you need ngModelOptions defined in the directive isolation scope. Also it would be nice if it were optional.

```
Cannot read property 'updateOn' of undefined
    at new NgModelOptionsController (gallery.js:95745)
```

So I dont have much time to look at this at the moment.

**Comment by ruhley** [comment]
@stephennancekivell I definitely tested without passing in `ng-model-options` and cannot replicate the problem. I may need more to go on to try and fix it.

**Comment by stephennancekivell** [comment]
Hi, yea thanks. It looks like its only on angular 1.5.

**Comment by ruhley** [comment]
Fixed in commit 61469d5545074ff3695e0966aad7e932ed5d6546 and released in [v3.4.3](https://github.com/ruhley/angular-color-picker/releases/tag/v3.4.3)

**Comment by stephennancekivell** [comment]
Legend! Thanks mate, I owe you a beer.
